TradeDis/backend
# Added:(backend-UserProfile-put) used for when user updates personal data

**What was done?**
Task: TRAD-71 Backend - PUT /users/{user_id}
Post model, service, controller, router and API configuration created
PUT /api/v1/users/{user_id}

**Testing**
- First, ensure the backend server is running (yarn dev)
- Testing PUT api/v1/users/{user_id} -> this is a variable so add any number

**Testing Steps:** - Using Postman
- First run a GET 'localhost:3000/api/v1/users' to get all users
- You should receive a JSON response of all users
- Testing PUT localhost:3000/api/v1/users/{user_id} -> pick a user_id to update and change any info 
- Then run a GET 'localhost:3000/api/v1/users/' to get all users and check if user updated
